addons/portal_rating/static/src/js/portal_chatter.js

```diff
@@ -294,13 +295,22 @@ PortalChatter.include({
         var messageIndex = $source.data("mes_index");
         var ratingId = this.messages[messageIndex].rating.id;
 
-        rpc('/website/rating/comment', {
-            "rating_id": ratingId,
-            "publisher_comment": '' // Empty publisher comment means no comment
-        }).then(function (res) {
-            self.messages[messageIndex].rating = self._preprocessCommentData(res, messageIndex);
-            self._getCommentButton($source).removeClass("d-none");
-            self._getCommentContainer($source).empty();
+        this.call("dialog", "add", ConfirmationDialog, {
+            title: _t("Delete confirmation"),
+            body: _t("Are you sure you want to permanently delete this comment?"),
+            confirm: () => {
+                rpc('/website/rating/comment', {
```

Comment:
:socks:  JS people are that kind of people, so try to use double quotes everywhere (at least don't mix them)